src/Kernel/ClassDefinitionPrinter.class.st

```diff
@@ -99,6 +99,11 @@ ClassDefinitionPrinter class >> for: aClass [
 		yourself
 ]
 
+{ #category : #testing }
+ClassDefinitionPrinter class >> isAbstract [
```

Comment:
Does this kind of definition inherited by the classes and should be overriden?

Reply:
Indeed, fixed